gemrb/plugins/FXOpcodes/FXOpcodes.cpp

```diff
@@ -1782,7 +1782,11 @@ int fx_morale_modifier (Scriptable* /*Owner*/, Actor* target, Effect* fx)
 {
 	if(0) print("fx_morale_modifier(%2d): Mod: %d, Type: %d", fx->Opcode, fx->Parameter1, fx->Parameter2);
 
-	//FIXME: in bg2 this is hacked to set param1=10, param2=1, we might need some flag for this
+	if (core->HasFeature(GF_FIXED_MORALE_OPCODE)) {
+		BASE_SET(IE_MORALE, 10);
```

Comment:
This makes it permanent, which I highly doubt was the intention. Would be better to just reset param1&2 (once, so check fx->FirstApply).

Reply:
I figured it's supposed to ignore timing mode as well, yes.  My evidence is `SPPR108.SPL` (Remove Fear), which has instant/permanent timing for this opcode.  If it respected timing, wouldn't this spell give fixed morale forever?

Reply:
Ah, indeed and the immunity effect that would block it is later in the queue. Odd spell, even more if you consider the description. FX_PERMANENT might be more semantically correct, but it doesn't matter at all.